go/teams/teams.go

```diff
@@ -175,13 +150,10 @@ func (t *Team) perUserEncryptionKey(ctx context.Context) (*libkb.NaclDHKeyPair,
 	if err := kr.Sync(ctx); err != nil {
 		return nil, err
 	}
-	encKey, err := kr.GetEncryptionKeyByKID(ctx, encKID)
+	encKey, err := kr.GetEncryptionKeyBySeqno(ctx, t.Box.PerUserKeySeqno)
 	if err != nil {
 		return nil, err
 	}
-	if encKey.Private == nil {
-		return nil, errors.New("per user enckey is locked")
-	}
```

Comment:
It should be impossible, the keyring's job is to store the secret. But checks are nice, I put it back.